docs/basics.md

```diff
@@ -238,7 +238,7 @@ Here is an example of backends and servers definition:
 
 # Launch
```

Comment:
>So # Launch becomes # Configuration, right ?

Yes

>But not sure to understand about add a section... 
Do you want me to add documentation about dynamic configuration here ?

Now that we can store static AND dynamic configuration into the kv-store, I think it would be better to explain a bit more the concept here.